README.md

```diff
@@ -0,0 +1,27 @@
+# AGA Ratings Program
+
+This program scans the AGA Database for tournaments that have been added but not yet rated.
+
+It is intended to be run after the tournaments have been uploaded into the database.
```

Comment:
I find this description using "this" and "it" a little off because we have many applications that are involved in getting our tournaments rated and added to our database. For example: are we running application with`$ bayrate`, `$ bayrate --commit` or `$ ./check`? Also, we have several binaries created:
```
aga@8939089dead4:/srv$ ls
basicrate  bayrate  check  copyMembership  syncDBs
```
by the Makefile. Some of which I am not sure exactly what they do. @vash3g Do you have information on these applications?

Reply:
copyMembership and SyncDBs are legacy and no longer needed. check and bayrate are part of the process in rating tournaments. check will go through and look for tournaments which have a "0" for the rated column in the games table and mark those for ratings. bayrate will perform the actual ratings for the application, first as a check. 'bayrate --commit' will actually rate the tournament and write to the database.

Reply:
Ok, cool, my current understanding, then, is that the usual workflow for this is something like the following. Correct me if I'm wrong please, otherwise I'll update the documentation to match. :)

1. Receive tournament data
2. <Run scripts outside of this repository, correcting errors>
3. Run `./check`
4. Run `./bayrate`
5. Run `./bayrate --commit`

Reply:
@vash3g we should consider removing "copyMembership" and "SyncDBs" from the AGA-Ratings-Program if they are no longer needed. (see: #9)

Reply:
@annabunches That sounds about right. Though, additionally on 2, I think we are partially committing a tournament to the AGAGD before we run `./check`.

Reply:
Added update to the readme file to include items listed in ToDo.

Adding PR to address issue #9 with updated makefile.